assets/scripts/ui.js

```diff
@@ -1,3 +1,56 @@
 'use strict';
+//Search by author
+const authorName = function(book) {
+  return book.author.toUpperCase() === $('#search').val().toUpperCase();
+};
+  const onSuccessAuthor = function(data) {
+  $('span').remove();
+  let books = (data.books.filter(authorName));
+  if(books.length > 0) {
+    for(let i = 0; i < books.length; i++) {
+    $('#insert').append('<span>' + $('#search').val() + " wrote " + books[i]["title"] + "<br /></span>");
```

Comment:
Instead of spans and breaks, next time consider an unordered list and use CSS to hide the bullet.